velox/functions/prestosql/Probability.h

```diff
@@ -61,5 +62,29 @@ struct NormalCDFFunction {
   }
 };
 
+template <typename T>
+struct BinomialCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, int64_t numOfTrials, double successProb, int64_t value) {
+    static constexpr int64_t kInf = std::numeric_limits<int64_t>::max();
+
+    VELOX_USER_CHECK(
+        (successProb >= 0) && (successProb <= 1),
+        "successProbability must be in the interval [0, 1]");
+    VELOX_USER_CHECK_GT(
+        numOfTrials, 0, "numberOfTrials must be greater than 0");
+
+    if ((value < 0) || (numOfTrials == kInf) || (value == kInf)) {
```

Comment:
So this the same behavior as Presto?

Reply:
@Yuhta Compared with presto and corrected logic to match results.